core/base/dimensionReduction/CMakeLists.txt

```diff
@@ -13,8 +11,11 @@ install(
   FILES
     ${CMAKE_CURRENT_SOURCE_DIR}/dimensionReduction.py
   DESTINATION
-    ${CMAKE_INSTALL_PREFIX}/scripts/ttk
+    "${CMAKE_INSTALL_DATAROOTDIR}/ttk"
```

Comment:
I don't believe this is the right place for scripts. on my system, that'd go under `/usr/local/share/scripts/ttk/` although I believe it should go to `/usr/local/scripts/ttk` (like other packages do, e.g. ospray)

Reply:
Done.